Marlin/language_de.h

```diff
@@ -146,6 +146,8 @@
 #define MSG_INIT_SDCARD                     "SD-Karte erkennen"  // Manually initialize the SD-card via user interface
 #define MSG_CNG_SDCARD                      "SD-Karte getauscht" // SD-card changed by user. For machines with no autocarddetect. Both send "M21"
 #define MSG_ZPROBE_OUT                      "Sensor ausserhalb"
+#define MSG_BLTOUCH_SELFTEST                "Bluetooth Test"
+#define MSG_BLTOUCH_RESET                   "Reset BLTouch"
```

Comment:
I think there is no difference, but your version sounds nicer, so I will take that